src/Integration.UnitTests/Binding/SolutionBindingOperationTests.cs

```diff
@@ -215,7 +170,7 @@ public void SolutionBindingOperation_CommitSolutionBinding()
 
         private SolutionBindingOperation CreateTestSubject()
         {
-            return new SolutionBindingOperation(serviceProvider, fileSystem);
```

Comment:
The `SolutionBindingOperation` only needed the service provider so it could get a `IProjectSystemHelper`. However, the method that used it is no longer called, so the constructor can be simplified.